### ISSUE TITLE
Terra Progressbar doesn't read current value accurately

### ISSUE THREAD

**Issue by JakeLaCombe** [cerner/terra-core#1838]
# Bug Report

## Description
Progress bars are leaving off the unit when they are read to the user, making it difficult to see what the progress bar is trying to say. To fix this, it's recommend to use aria-valuetext as a prop on the progress bar.

## Additional Context / Screenshots
For color blindness, it's more ideal to use textures instead of straight up color. Perhaps consider using textures for some cases? That maybe better left to another Jira.
